.openshift-ci/build/Dockerfile.build_collector

```diff
@@ -0,0 +1,6 @@
+FROM will_be_replaced_by_openshift_ci
+
+ENV SRC_ROOT_DIR=/go/src/github.com/stackrox/collector
+WORKDIR $SRC_ROOT_DIR
+
+RUN ./.openshift-ci/build/build-collector.sh $SRC_ROOT_DIR
```

Comment:
Passing environment variable SRC_ROOT_DIR into the script and then re-exporting in the script is redundant.